docs/tutor.rst

```diff
@@ -87,3 +87,23 @@ Third-party contributions to Tutor and its plugins are more than welcome! Just m
 Where "TYPE" is either "Bugfix", "Improvement", "Feature" or "Security". You should add an explosion emoji ("💥") before "[TYPE]" if you are making a breaking change.
 
 Happy hacking! ☘️
+
+Becoming a Tutor Maintainer
+---------------------------
+
+Is Tutor a core part of your infrastructure? Would you like to contribute to the project roadmap? Then you should consider becoming a Tutor maintainer. Tutor maintainers are individuals and companies who help make decisions on the future of the prject. Their goal is to support the Tutor community and to make the software as best as possible.
```

Comment:
Nit: s/prject/project/

Reply:
Also, _anyone_ can become a maintainer? Imho having some kind of selection from you @regisb would allow to give more meaning to the role of maintainer - ie make it more of a reward/badge granted for work done. It would make it something people would work towards, and that you can give as a mark of appreciation and recognition. It would also allow you to build a team of dedicated people that you trust, which would make it easier to consider delegating work to that group - including things you might not want to hand out to anyone knocking on the door?

Reply:
Yes, anyone can become a maintainer. I do not want to create early a barrier to entry. I figure, if someone applies then they believe they have the required expertise. Of course, I will interview everyone personally and we will discuss which responsibilities best match their skillset, available time and interest. For instance, only developers will be able to review pull requests.